app/views/courses/CoursesView.coffee

```diff
@@ -43,6 +46,16 @@ module.exports = class CoursesView extends RootView
     @courses = new CocoCollection([], { url: "/db/course", model: Course})
     @supermodel.loadCollection(@courses)
 
+    # TODO: Trim this section for only what's necessary
+    @hero = new ThangType
+    defaultHeroOriginal = ThangType.heroes.captain
+    heroOriginal = me.get('heroConfig')?.thangType or defaultHeroOriginal
+    @hero.url = "/db/thang.type/#{heroOriginal}/version"
+    # @hero.setProjection ['name','slug','soundTriggers','featureImages','gems','heroClass','description','components','extendedName','unlockLevelName','i18n']
+    @supermodel.loadModel(@hero, 'hero')
```

Comment:
`loadModel` and `loadCollection` are deprecated. Use instead

`supermodel.track(@hero.fetchLatestVersion(heroOriginal))`

Where `fetchLatestVersion` is a standard fetch-type function in `CocoModel`, since it's a function used in many collections.